### ISSUE TITLE
About repeat delay !

### ISSUE THREAD

**Issue by NeximaOzi** [JCulley/ProQuests-Documentation#128]
Hello again..
Spigot name: Nexima
Spigot Account Link: https://www.spigotmc.org/members/nexima.40441/
ProQuest version: Latest
Spigot version: 1.12.2
Log: https://hastebin.com/oholikepuj.md
A Quest: https://hastebin.com/kilusapaci.bash
Config: https://hastebin.com/rogohacoje.php

I set "repeat_delay: 86400" (1 day) for all quests.
For example a player get a quest and finish this quest when going to main lobby. Again join to faction server and again can get same quest. So don't wait 1 day.

 I don't use mysql so what can i do for this ?
I'm not opened useless tickets. I just want help for this issue.

Repeat delay options not working for bungee servers. Because when players going to main lobby and joined again server, again can get same quest not waiting time..

**Comment by JCulley** [comment]
You continue to provide no additional information that is of use to me. Do not open another issue unless you’re going to provide sufficient information. Thanks.

**Comment by NeximaOzi** [comment]
So what you want for information ?
